packages/langium/src/test/langium-test.ts

```diff
@@ -231,14 +231,11 @@ export interface ExpectDiagnosticOffsetOptions {
 
 export type Predicate<T> = (arg: T) => boolean;
 
-function isLeftOrEqualFrom(lhs: Position, rhs: Position) {
-    return lhs.line < rhs.line || lhs.line === rhs.line && lhs.character <= rhs.character;
-}
-
-function containsRange(outer: Range, inner: Range): boolean {
-    const start = isLeftOrEqualFrom(outer.start, inner.start);
-    const end = isLeftOrEqualFrom(inner.end, outer.end);
-    return start && end;
+function isRangeEqual(lhs: Range, rhs: Range): boolean {
+    return lhs.start.character == rhs.start.character
```

Comment:
You should install the `eslint` extension for vscode to catch those during dev-time.